marklogic-data-hub/src/test/resources/pii-test/entity-config/final-entity-options.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not really needed for this test.  I think you should remove it so as to make what you're testing cleaner.